packages/cli/src/commands/deviceprofiles/translations/upsert.ts

```diff
@@ -0,0 +1,74 @@
+import {DeviceProfile, DeviceProfileTranslations} from '@smartthings/core-sdk'
+import {SelectingInputOutputAPICommand} from '@smartthings/cli-lib'
+
+import {buildTableOutput} from '../translations'
+
+
+export default class CapabilityTranslationsUpsertCommand extends SelectingInputOutputAPICommand<DeviceProfileTranslations, DeviceProfileTranslations, DeviceProfile> {
```

Comment:
This should be ~Capability~ DeviceProfileTranslationsUpsertCommand.

Reply:
Fixed